### PR TITLE
Navigation manager UI enhancement

### PR DESCRIPTION
**Pull request by TheImpressiveDonut**

### CODE REVIEW

File: app/src/main/java/com/github/onedirection/navigation/fragment/map/MapFragment.java [hunk 1]
```diff
@@ -101,6 +101,9 @@ public View onCreateView(@NonNull LayoutInflater inflater,
             mapboxMap.setStyle(Style.MAPBOX_STREETS, style -> {
                 initializeDeviceLocationProvider();
                 initializeManagers(style, view);
+                /*LatLng LAT_LNG_1 = new LatLng(37.423018, -122.083361);
+                LatLng LAT_LNG_2 = new LatLng(37.432067, -122.087706);
+                routesManager.findRoute(LAT_LNG_1, Collections.singletonList(LAT_LNG_2), new NavigationRouteResponseListener());*/
```
Comment (Ef55) [inline app/src/main/java/com/github/onedirection/navigation/fragment/map/MapFragment.java:106]: Should most likely be gone.

**Review by Ef55 — CHANGES_REQUESTED** [review]
All seems good, except

- A few linter warnings;
- The test coverage is a bit low (yeah, I'm one to talk).

**Review by hugolan — APPROVED** [review]
Good for me, check the lint warnings

File: app/src/main/java/com/github/onedirection/navigation/fragment/map/MapFragment.java (hunk 1 above)
Comment (TheImpressiveDonut) [reply app/src/main/java/com/github/onedirection/navigation/fragment/map/MapFragment.java:106]: Yes

**Review by Ef55 — APPROVED** [review]

**Pull request merged**